src/assets/server/rest/v1/docs/e-mobility-oas.json

```diff
@@ -5386,6 +5375,14 @@
           "type": "boolean"
         }
       },
+      "RefundStatus": {
+        "in": "query",
+        "name": "RefundStatus",
+        "description": "Pipe separated refund statuses (submitted, notSubmitted, cancelled, approved)",
+        "schema": {
+          "type": "string"
```

Comment:
I think you can enforce the string list.

Reply:
I could add pattern check here, but is it relevant as it is only documentation ?

Reply:
The code strictly enforce the list to that enum: 

```ts
export enum RefundStatus {
  SUBMITTED = 'submitted',
  NOT_SUBMITTED = 'notSubmitted',
  CANCELLED = 'cancelled',
  APPROVED = 'approved',
}
```

So must do the inputs validator.

Reply:
You're right, haven't noticed validators patterns were missing for multiple values. 
Done.